server/src/assets/game/driver-game.html

```diff
@@ -222,8 +236,14 @@ <h5 id="serverModalTitle" class="modal-title"></h5>
 	<form id="destinationsForm">
 		<label class="col-form-label-lg me-md-5">
 			<b>
-				<span lang="de">Mögliche Ziele:</span>
-				<span lang="en">Possible destinations:</span>
+				<span lang="de">
+					<span class='easy'>Klicke auf das Symbol, zu dem du fahren möchtest: </span>
+					<span class='normal'>Mögliche Ziele: </span>
+				</span>
+				<span lang="en">
+					<span class='easy'>Click the symbol you wish to drive to: </span>
+					<span class='normal'>Possible destinations:</span>
```

Comment:
Am Ende mancher Span-Inhalte ist ein Leerzeichen, aber nicht bei allen. Sollte konsistent sein. Betrifft auch andere Spans weiter unten.
At the end of some span-contents there is a space, but not in all cases. This should be made consistent if possible. Also check other spans in this file.